src/System.Diagnostics.PerformanceCounter/src/PinvokeAnalyzerExceptionList.analyzerdata.netstandard

```diff
@@ -0,0 +1,4 @@
+kernel32.dll!CreateFileMappingW
```

Comment:
@adiaaida you can delete this file now, since you have `<UWPCompatible>false</UWPCompatible>` in the csproj, it's not going to complain that these API aren't available on UWP anymore.